src/components/LayoutTree.tsx

```diff
@@ -139,6 +153,7 @@ const LayoutTree = () => {
 
       setLayoutTop(ref.current?.getBoundingClientRect().top ?? 0)
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
When disabling the linter, I like to include a comment that explains what dependency is being ignored and why. This helps to discourage developers from disabling the linter unnecessarily.